harvard-lts/drs-translation-service
# Etd 234

**Add content models for Documents (and others).**
* * *

**GitHub Issue**: [(link)](https://jira.huit.harvard.edu/browse/ETD-234)
Other jiras
https://jira.huit.harvard.edu/browse/ETD-235 (Text)
https://jira.huit.harvard.edu/browse/ETD-236 (Still Image)
https://jira.huit.harvard.edu/browse/ETD-237 (Audio)

# What does this Pull Request do?
This pr adds content models for Documents, Text, Still Image and Audio. Note, originally it was meant for just Documents, but following the [documentation](https://docs.google.com/document/d/1TqhWG2wCz5pKvTLfhFGPINz6VVmCOlBgQc6vv2Ua4nM/edit#heading=h.a0jjorckudrg), I inadvertently jumped to doing all. I believe these are all following the same model, but I can back the latter 3 out if there it adds confusion.

# How should this be tested?

Check out this branch
Build Docker: docker-compose -f docker-compose-local.yml up -d --build --force-recreate
Exec into container: docker exec -it dts bash
Run pytest, all, including 4 new tests for content models, should pass

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? Y
- integration tests?

# Interested parties
Tag (@ mention) interested parties
